ocs_ci/utility/ocs_build.py

```diff
@@ -0,0 +1,67 @@
+"""
+This module is used to return latest OCS internal build for specified OCS
+version.
+"""
+
+import argparse
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.framework.main import load_config
+from ocs_ci.ocs.constants import OCS_VERSION_CONF_DIR
+from ocs_ci.utility.utils import get_latest_ds_olm_tag
+
+
+def init_parser():
+    """
+    Init argument parser.
+
+    Return:
+        object: Parsed arguments
+    """
+
+    parser = argparse.ArgumentParser(
+        description="OCS Internal build version",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "--ocs-version",
+        action="store",
+        required=False,
+        default=config.ENV_DATA["ocs_version"],
+        help=f"""
+            OCS version in format X.Y (e.g. 4.7). If not specified, the default
+            value {config.ENV_DATA['ocs_version']} will be used.
+        """,
+    )
+    parser.add_argument(
+        "--image",
+        action=argparse.BooleanOptionalAction,
```

Comment:
This need python 3.9 version. ( may be its worth to mention in usage of build tool )

Reply:
updated to support older version as well